samples/net/aws_iot/README.rst

```diff
@@ -0,0 +1,255 @@
+Note that the *modem_version* parameter in messages published to AWS IoT will not be present for Wi-Fi builds.
+
+.. code-block:: console
+
+   *** Booting Zephyr OS build v3.3.99-ncs1-12-g2f5e820792d1 ***
+   [00:00:00.253,204] <inf> aws_iot_sample: The AWS IoT sample started, version: v1.0.0
+   [00:00:00.253,234] <inf> aws_iot_sample: Bringing network interface up and connecting to the network
+   [00:00:03.736,450] <inf> aws_iot_sample: IP Up
+   [00:00:08.736,572] <inf> aws_iot_sample: Connecting to AWS IoT
+   [00:00:08.736,633] <inf> aws_iot_sample: Next connection retry in 30 seconds
+   [00:00:08.736,663] <inf> aws_iot_sample: AWS_IOT_EVT_CONNECTING
+   [00:00:12.855,072] <inf> aws_iot_sample: AWS_IOT_EVT_CONNECTED
+   [00:00:12.856,323] <inf> aws_iot_sample: Publishing message: {"state":{"reported":{"uptime":12855,"app_version":"v1.0.0","modem_version":"nrf9160_1.3.4"}}} to AWS IoT shadow
+   [00:00:13.228,881] <inf> aws_iot_sample: AWS_IOT_EVT_READY
+   [00:00:13.244,781] <inf> aws_iot_sample: AWS_IOT_EVT_PUBACK, message ID: 32367
+   [00:01:12.867,675] <inf> aws_iot_sample: Publishing message: {"state":{"reported":{"uptime":72858,"app_version":"v1.0.0","modem_version":"nrf9160_1.3.4"}}} to AWS IoT shadow
+   [00:02:12.883,789] <inf> aws_iot_sample: Publishing message: {"state":{"reported":{"uptime":132874,"app_version":"v1.0.0","modem_version":"nrf9160_1.3.4"}}} to AWS IoT shadow
+
+.. note::
+   For nRF91 series boards the output will differ from the example output listed above.
+   This is because the sampe enables the :ref:`lib_at_host` library via the :kconfig:option:`CONFIG_AT_HOST_LIBRARY` option.
+   This library makes it possible to send and receive AT commands to and from the cellular modem using the nRF Connect for Desktop LTE Link monitor and Cellular Monitor applications.
+   The additional logs are AT command responses that the modem sends to the application core that is forwarded over UART to be picked up by any of the aforementioned Desktop applications.
+
+To observe incoming messages in the AWS IoT console, follow the steps documented in :ref:`aws_iot_testing_and_debugging`.
+
+Emulation
+=========
+
+The sample can be run in :ref:`QEMU x86 <zephyr:qemu_x86>` that simplifies development and testing and removes the need for hardware.
+Before you can build and run on QEMU, you need to perform the steps included in this link: :ref:`networking_with_qemu`.
+
+When the aforementioned steps are completed, you can build and run the sample by using the following commands:
+
+.. code-block:: console
+
+   west build -b qemu_x86 samples/net/mqtt
+   west build -t run
+
+Troubleshooting
+===============
+
+To enable more verbose logging from the AWS IoT library, set the :kconfig:option:`CONFIG_AWS_IOT_LOG_LEVEL_DBG` option.
+
+* If you are having issues with connectivity on nRF91 series boards, see the `Trace Collector`_ documentation to learn how to capture modem traces in order to debug network traffic in Wireshark.
```

Comment:
If the Cellular Monitor docs are live, we should probably refer to them instead

Reply:
I haven't been able to find them, ill have a second look.

Reply:
I think we should drop the link to the Trace collector as it is deprecated, and then maybe just mention Cellular Monitor now and link to it when the docs are live

Reply:
Or, just do a tree wide change when the time comes. AFAIC the trace collector is still an applocation in nrf connect for desktop?